.github/workflows/build.yml

```diff
@@ -304,7 +272,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: liquibase-zip-${{ needs.setup.outputs.thisBranchName }}
-          path: liquibase-dist/target/liquibase-${{ needs.setup.outputs.thisBranchName }}-SNAPSHOT/*
+          path: artifacts/*.zip
```

Comment:
This is causing the zip to contain another directory, which is breaking functional tests.

Reply:
Fixed by extracting the tarball to a directory and uploading that instead. This is basically what happened before, without all the `diff`-ing.

Reply:
That sucks